sparse_strips/vello_cpu/tests/issues.rs

```diff
@@ -0,0 +1,285 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Tests for Github issues.
+
+use crate::util::{check_ref, get_ctx, render_pixmap};
+use vello_common::color::palette::css::{DARK_BLUE, LIME};
+use vello_common::kurbo::{BezPath, Stroke};
+use vello_common::peniko::Fill;
+
+mod util;
+
+// All issue numbers starting from here refer to the issue numbers in the old repo
+// at https://github.com/LaurenzV/cpu-sparse-experiments/issues.
+
+#[test]
+fn issue_2_incorrect_filling_1() {
```

Comment:
I don't think test names referring to issue numbers from a different repository are helpful.
I'd suggest either just `incorrect_filling_1` or maybe `rotated_square`?

It's definitely good to have regression tests, but a lot of these are testing baseline functionality, and it's just a historical coincidence that they had corresponding issues.